README.md

```diff
@@ -21,29 +21,31 @@ This sample uses the [Microsoft Authentication Library Preview for JavaScript (m
 ## Prerequisites
 
 To use the Microsoft Graph Connect sample for AngularJS, you need the following:
-* [Node.js](https://nodejs.org/). Node is required to run the sample on a development server and to install dependencies. 
+* [Node.js](https://nodejs.org/). Node is required to run the sample on a development server and to install dependencies.
 
 * Either a [Microsoft account](https://www.outlook.com) or [Office 365 for business account](https://msdn.microsoft.com/en-us/office/office365/howto/setup-development-environment#bk_Office365Account)
 
 ## Register the application
 
-1. Sign into the [App Registration Portal](https://apps.dev.microsoft.com/) using either your personal or work or school account.
+1. Sign in to the [Azure Portal](https://portal.azure.com/) to register an application.
 
-2. Choose **Add an app**.
+1. If your account gives you access to more than one tenant, select your account in the top right corner, and set your portal session to the desired Azure AD tenant.
 
-3. Enter a name for the app, and choose **Create application**. 
-	
-   The registration page displays, listing the properties of your app.
+1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations (Preview) > New registration**.
 
-4. Copy the Application Id. This is the unique identifier for your app. 
+1. When the **Register an application** page appears, enter a name for your application.
 
-5. Under **Platforms**, choose **Add Platform**.
+1. Under **Supported Microsoft accounts**, select **Accounts in any organizational directory and personal Microsoft accounts**.
```

Comment:
"Supported Microsoft accounts" changed to "Supported account types"

Reply:
This "Accounts in any organizational directory and personal Microsoft accounts**." option no longer shows up anymore, Replace with "Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)"

Reply:
Thank you for checking. Updated accordingly.